frontend/src/scenes/Workspace/sagas/index.ts

```diff
@@ -23,6 +29,50 @@ function* watchPostWorkspaceName() {
   yield takeEvery(addWorkspaceRoutine.TRIGGER, addWorkspaceReq);
 }
 
+function* addCommentRequest({ payload }: Routine<any>) {
```

Comment:
it should be on src/containers/Thread level

Reply:
Add comment request now in src/containers/Thread